src/cljstyle/format/indent.clj

```diff
@@ -120,9 +120,15 @@
   (-> zloc z/leftmost* margin))
 
 
+(defn- repeated-move
+  "Repeatedly move a direction"
+  [zloc move n]
+  (nth (iterate move zloc) n))
```

Comment:
This is going to get called quite frequently during the code walk; could you switch this to use a loop instead? The overhead of lazy sequence thunks is small but adds up.

Reply:
Good call, thanks